src/env/start.rs

```diff
@@ -71,7 +71,7 @@ pub async fn execute(terminate: Arc<AtomicBool>) -> anyhow::Result<Environment>
     );
     envfile.update(
         "ERC20_CONTRACT_ADDRESS",
-        &format!("0x{:x}", parity.erc20_contract_address),
+        &format!("{:#x}", parity.erc20_contract_address),
```

Comment:
Do we have a test for that?

Reply:
Not yet but there is a ticket: https://github.com/comit-network/create-comit-app/issues/94